espnet2/layers/houlsby_adapter_layer.py

```diff
@@ -0,0 +1,123 @@
+import torch
+import torch.nn as nn
+
+try:
+    import s3prl
+    from s3prl.upstream.wav2vec2.wav2vec2_model import TransformerSentenceEncoderLayer
+
+    is_s3prl_available = True
+except ImportError:
+    is_s3prl_available = False
+
+
+class Houlsby_Adapter(nn.Module):
+    def __init__(
+        self,
+        input_size: int,
+        bottleneck: int,
+    ):
+        super(Houlsby_Adapter, self).__init__()
+        self.bottleneck = bottleneck
+        self.houlsby_adapter = nn.Sequential(
+            nn.Linear(input_size, self.bottleneck),
+            nn.GELU(),
+            nn.Linear(self.bottleneck, input_size),
+        )
+
+    def forward(self, x):
+        return self.houlsby_adapter(x)
+
+
+if not is_s3prl_available:
+    HoulsbyTransformerSentenceEncoderLayer = None
+else:
+
+    class HoulsbyTransformerSentenceEncoderLayer(TransformerSentenceEncoderLayer):
```

Comment:
It seems that this class does not go through the test.
Can you check it?

Reply:
May you check this now?